### ISSUE TITLE
NullReferenceException

### ISSUE THREAD

**Issue by zazizazi** [phoenixzz/IvyGenerator#1]
Getting a NullReferenceException at IvyEditor.cs:280
any clue?

```
NullReferenceException: Object reference not set to an instance of an object
Ivy.IvyEditor.PlaceRoot (Vector3 placePos) (at Assets/Ivy/Scripts/Editor/IvyEditor.cs:280)
Ivy.IvyEditor.OnSceneGUI () (at Assets/Ivy/Scripts/Editor/IvyEditor.cs:66)
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:222)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:232)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MethodBase.cs:115)
UnityEditor.SceneView.CallOnSceneGUI () (at C:/buildslave/unity/build/Editor/Mono/SceneView/SceneView.cs:2386)
UnityEditor.SceneView.HandleSelectionAndOnSceneGUI () (at C:/buildslave/unity/build/Editor/Mono/SceneView/SceneView.cs:1715)
UnityEditor.SceneView.OnGUI () (at C:/buildslave/unity/build/Editor/Mono/SceneView/SceneView.cs:1553)
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:222)
```

**Comment by phoenixzz** [comment]
What actions do you get this Exception? It seems 'script' variable in function OnInspectorGUI() is null. This function is invoked by Unity Editor, therefore it seems never be null (target must be IvyManager).
What is the version of Unity? Do you generate the "Ivy manager" by Menu "Window" -> "Ivy Generation System"->"Ivy Manager"?

**Comment by zazizazi** [comment]
The OS is Win 8.1x64 with Unity 5.6.0f3 x32 Personal.
I followed the instructions, basically :
1. Created fresh 3d project
2. Imported the IvyGenerator.unitypackage
3. Created a Cube GameObject in the scene, made sure it has a box collider
4.Created Ivy Manager GO from Window->Ivy Generation System->Ivy Manager
5. Then I pushed the Start Plant button on the Ivy Manager, pointed the mouse on the Cube in the scene and hit 'p' on the keyboard.
And voila - I get the Exeption.

I have looked at the code and at first glance it seems that this should not happen, because of the reasons you've mentioned.
